### PR TITLE
feat: support wtns generation

### PR DESCRIPTION
**Pull request by sorawee**
The commit adds --wtns, which indicates the output directory for the witness files. If not given, no witness file is generated. The witness files are named `first-witness.wtns` and `second-witness.wtns`.

### CODE REVIEW

File: picus/algorithms/dpvl.rkt [hunk 1]
```diff
@@ -260,66 +260,75 @@
...
+      ;; where x indicates that it is in the first set
+      ;; and y indicates that it is in the second set.
+      [(pregexp #px"^(?:x|y)(\\d+)$" (list _ (app string->number n)))
+       (values input-info
+               output1-info
+               output2-info
+               (if (string-prefix? k "x") (cons (cons n v) other1-info) other1-info)
+               (if (string-prefix? k "y") (cons (cons n v) other2-info) other2-info))]
+
+      [(pregexp #px"^(?:ps\\d|zero|one|p)$")
```
Comment (shankarapailoor) [inline picus/algorithms/dpvl.rkt:317]: What is this regex looking for?

Comment (sorawee) [reply picus/algorithms/dpvl.rkt:317]: It looks for symbolic variables that are inserted by Picus, but not defined by the circuit itself. I can add a comment explaining that.

Comment (shankarapailoor) [reply picus/algorithms/dpvl.rkt:317]: That would be great thanks!

**Review by shankarapailoor — APPROVED** [review]
LGTM

**Pull request merged**